docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -22,19 +22,19 @@ showAsideToc: true
   </li>
 </ul>
 
-
 Row-level geo-partitioning allows fine-grained control over pinning data in a user table (at a per-row level) to geographic locations, thereby allowing the data residency to be managed at the database level. Use-cases requiring low latency multi-region deployments, transactional consistency semantics and transparently schema change propagation across the regions would benefit from this feature. 
```

Comment:
The first line is a bit confusing... We say row level geo partitioning and then say data residency managed at database level... Do you know what we mean to say here?